scripts/build/libc/glibc.sh

```diff
@@ -167,7 +175,8 @@ do_libc_backend() {
                              libc_headers="${libc_headers}"         \
                              libc_startfiles="${libc_startfiles}"   \
                              libc_full="${libc_full}"               \
-                             libc_target="${target}"
+                             libc_target="${target}"                \
```

Comment:
No, it sets ${target} above but passes it down as ${libc_target}.
It's mingwandroid's original change, I picked it up as is (mostly, so long as that worked :)) This code was later changed into a common iterator anyway.